tests/set-labels/legacy-config/fn-config.yaml

```diff
@@ -1,13 +1,7 @@
 apiVersion: fn.kpt.dev/v1alpha1
-kind: SetLabelConfig
+kind: SetLabels
```

Comment:
yeah, we discussed this and we will remove the tests in the following up PR. Because this PR refactors the entire function and we do want to make sure it is under good test coverage. 
 Here's the follow-up work https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/878#issuecomment-1167671033

Reply:
Got it.